client/dashboard/me/notifications-extras/extras-toggle-card/test/index.test.tsx

```diff
@@ -0,0 +1,239 @@
+/**
+ * @jest-environment jsdom
+ */
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { WPCOM_OPTION_KEYS, WPCOM_TITLES, WPCOM_DESCRIPTIONS } from '../../extras-config';
+import { ExtrasToggleCard } from '../index';
+import type { WpcomNotificationSettings } from '@automattic/api-core';
+
+const mockOnMutate = jest.fn();
+
+const defaultExtraSettings: Partial< WpcomNotificationSettings > = {
+	marketing: false,
+	research: false,
+	community: false,
+	promotion: false,
+	news: false,
+	digest: false,
+	reports: false,
+	news_developer: false,
+	scheduled_updates: false,
+};
+
+describe( 'ExtrasToggleCard', () => {
+	beforeEach( () => {
+		mockOnMutate.mockClear();
+	} );
+
+	const renderExtrasToggleCard = ( props = {} ) => {
+		return render(
+			<ExtrasToggleCard
+				extraSettings={ defaultExtraSettings }
+				isSaving={ false }
+				onMutate={ mockOnMutate }
+				optionKeys={ WPCOM_OPTION_KEYS }
+				titles={ WPCOM_TITLES }
+				descriptions={ WPCOM_DESCRIPTIONS }
+				sectionTitle="Test Section"
+				{ ...props }
+			/>
+		);
+	};
+
+	it( 'renders section title', () => {
+		renderExtrasToggleCard();
```

Comment:
```suggestion
		renderExtrasToggleCard({sectionTitle: "Some title");
```
It creates a connection between the test and the component usage, similar we have in the scenario below ("renders section description when provided")

Reply:
Updated. Its essentially doing the exact same thing, but now just overriding the value and checking for a different one. But to make the test slightly more readable by adding the value we should expect within the test itself as opposed to the default test rendering definition.